.env.example

```
@@ -0,0 +1,32 @@
+# Local Auth serv
+REEARTH_AUTH0_DOMAIN=http://localhost:8080
+REEARTH_AUTH0_AUDIENCE=http://localhost:3000
+REEARTH_AUTH0_CLIENTID=reearth-authsrv-client-default
+REEARTH_AUTH0_CLIENTSECRET=
+REEARTH_AUTH0_WEBCLIENTID=
```

```suggestion
REEARTH_AUTH0_DOMAIN=https://example.auth0.com
REEARTH_AUTH0_AUDIENCE=https://api.reearth.example.com/
REEARTH_AUTH0_CLIENTID=
REEARTH_AUTH0_CLIENTSECRET=
REEARTH_AUTH0_WEBCLIENTID=
```

localhost will not be used for Auth0 domain in backend